pubsub/spec/spec_helper.rb

```diff
@@ -13,3 +13,8 @@
 # limitations under the License.
 
 require "rspec"
+require "rack/test"
+
+RSpec.configure do |conf|
+  conf.include Rack::Test::Methods
```

Comment:
It's conventional to include modules such as this in the `_spec.rb` describe block.

```ruby
describe "Pub/Sub samples" do
  include Rack::Test::Methods

  # ...
end
```